chained/tlsmasq_impl.go

```diff
@@ -73,7 +69,7 @@ func newTLSMasqImpl(configDir, name, addr string, pc *config.ProxyConfig, uc com
 		return nil, errors.New("malformed server address: %v", err)
 	}
 
-	proxyTLS, hellos, err := tlsConfigForProxy(ctx, configDir, name, pc, uc)
+	proxyTLS, hellos, err := tlsConfigForProxy(context.Background(), configDir, name, pc, uc)
```

Comment:
why was the timeout removed?

Reply:
This was the result of the bug I mentioned on slack where all of the hellobrowser-based transports were not picking up the clienthello successfully, so each one would result in a 5 second delay getting connected. If you have 6 proxies, that would be a 30 second delay because it would wait for all of them when creating the dialers map.